main/tests/server/src/com/google/refine/clustering/binning/KeyerTests.java

```diff
@@ -100,6 +100,9 @@
 //         "bbddddhiikkllmnnooooopprssssttttuuuuyyzz                      aaiioouuuuuuuuuux"},
 //        {"ǞǟǠǡǢǣǤǥǦǧǨǩǪǫǬǭǮǯǰǱǲǳǴǵǶǷǸǹǺǻǼǽǾǿȀȁȂȃȄȅȆȇȈȉȊȋȌȍȎȏȐȑȒȓȔȕȖȗȘșȚțȜȝȞȟȠȡȢȣȤȥȦȧȨȩȪȫȬȭȮȯȰȱȲȳȴȵȶȷȸȹȺȻȼȽȾȿɀɁɂɃɄɅɆɇɈɉɊɋɌɍɎɏ",
 //         "aaaaaeaeggggkkqqqqqssjdzdzggpnnaaaeaeooaaaeeeeeiiiioooorrrruuuusstt    zzaaeeooooooooyy"},
+        // WIP: These we know should be folded based on research (Wikipedia,etc.), and modern expectations (transliterations) by researchers (philologists)
+        {"Ƿƿ",
+         "ww"},
```

Comment:
```suggestion
        {"Ƿƿ", "ww"}, // Old English letter wynn
```
Let's also relocate this up below the test for eth and thorn since they're similar.